build.sbt

```
@@ -238,7 +238,7 @@ lazy val interfaces = project
     moduleName := "mtags-interfaces",
     autoScalaLibrary := false,
     mimaPreviousArtifacts := Set(
-      "org.scalameta" % "mtags-interfaces" % "1.0.1"
+      "org.scalameta" % "mtags-interfaces" % "1.2.1"
```

```suggestion
      "org.scalameta" % "mtags-interfaces" % "1.2.0"
```
for now I guess and we can update it later on.